web/themes/custom/cloudy/templates/layout/page.html.twig

```diff
@@ -34,62 +34,110 @@
+            {% set header_form = page.header_form|render %}
+            {% if header_form %}
+              <div class="form-inline navbar-form">
+                {{ page.header_form }}
+              </div>
+            {% endif %}
+          </div>
+        {% endif %}
+      </div>
+    </nav>
+  </header>
+{% endblock %}
+
+{% block tabs %}
+  {{ page.tabs }}
+{% endblock %}
+
+{% block messages %}
+  {{ page.messages }}
+{% endblock %}
+
+{% block pre_content %}
+  {{ page.pre_content }}
+{% endblock %}
+
+{% set content_wrapper_attribute = create_attribute() %}
+{% set content_wrapper_classes = [
+    not is_front ? 'mt-4',
+    page.tools or page.menu ? 'row'
   ]
 %}
-<div {{ attributes.addClass(container_classes) }} data-swiftype-index="true">
-  <div class="row">
-    {{ page.content }}
-    {% if page.tools %}
-      {{ page.tools }}
-    {% endif %}
-    {% if page.menu %}
-      {{ page.menu }}
-    {% endif %}
-  </div>
-</div>
-<footer class="site-footer text-dark">
-  <div class="container">
-    <div class="row">
-      {% if page.footer_first or page.footer_second %}
-        <div class="col-sm-6" id="footer_first">
-          {{ page.footer_first }}
-        </div>
-        <div class="col-sm-6" id="footer_second">
-          {{ page.footer_second }}
+
+{% set content_attribute = create_attribute() %}
+{% set content_classes = [
+    'page-content',
+    page.tools or page.menu ? 'col',
+    page.menu ? 'order-lg-2'
+  ]
+%}
+
+{% set aside_attribute = create_attribute() %}
+{% set aside_classes = [
+    'col-sm-12',
+    'col-lg-3',
+    'order-1'
+  ]
+%}
+
+<main id="main-content" role="main" tabindex="-1" data-swiftype-index="true">
+  <section>
+    {% block main_content %}
+      <div class="main-content {{ not is_front ? 'container' }}">
+        <div {{ content_wrapper_attribute.addClass(content_wrapper_classes) }}>
+          {% if page.menu %}
+            <aside {{ aside_attribute.addClass(aside_classes, 'sidebar-left')}}>
+              {{ page.menu }}
+            </aside>
+          {% endif %}
+          <div {{ content_attribute.addClass(content_classes)}}>
+            {% block page_content %}
+              {{ page.content }}
+            {% endblock %}
+          </div>
+          {% if page.tools %}
+            <aside {{ aside_attribute.addClass(aside_classes, 'order-md-2 sidebar-right')}}>
+              {{ page.tools }}
+            </aside>
+          {% endif %}
         </div>
-      {% endif %}
+      </div>
+    {% endblock %}
+  </section>
+</main>
+
+{% block footer %}
+  <footer class="site-footer text-dark">
+    <div class="container">
+      <div class="row">
+        {% if page.footer_first or page.footer_second %}
+          <div class="col-sm-6" id="footer_first">
+            {{ page.footer_first }}
+          </div>
+          <div class="col-sm-6" id="footer_second">
+            {{ page.footer_second }}
+          </div>
+        {% endif %}
+      </div>
     </div>
-  </div>
```

Comment:
I don't have this extra div in my local. Wonder where that came from. 🤔

Reply:
hmm, not sure.